### PR TITLE
:sparkles: bring back the overall graph

### PR DESCRIPTION
**Pull request by pNagi**

### CODE REVIEW

File: src/shared/nt-results/ResultsByLoadContainer.react.js [hunk 1]
```diff
@@ -97,30 +119,37 @@ class ResultsByLoadContainer extends React.Component {
...
   )
 
+  renderABGraphs = () => ([
+    this.renderResults('graphA', 'nt__A', { width: `${this.state.x}px` }),
+    <div
+      key='divider'
+      className={styles.nt__divider}
+      style={{ transform: `translateX(${this.state.x}px)` }}
+      ref={c => { this.divider = c }}
+    />,
```
Comment (0xchimz) [inline src/shared/nt-results/ResultsByLoadContainer.react.js]: `renderABGraphs` return an array without unique `key` of each element.

**Review by 0xchimz — CHANGES_REQUESTED** [review]

File: src/shared/nt-results/ResultsByLoadContainer.react.js [hunk 2]
```diff
@@ -122,9 +122,10 @@ class ResultsByLoadContainer extends React.Component {
   renderResults = (graph, styleName, style) => (
     <div className={styles[styleName]} style={style}>
       {this.props.data.loading ? (
-        <Loading />
+        <Loading key='loading' />
```
Comment (chinclubi) [inline src/shared/nt-results/ResultsByLoadContainer.react.js]: key is not unique.
what's about `key='loading-${graph}'`?

**Review by 0xchimz — APPROVED** [review]

File: src/shared/nt-results/ResultsByLoadContainer.react.js [hunk 3]
```diff
@@ -97,30 +119,38 @@ class ResultsByLoadContainer extends React.Component {
     })
   }
 
-  renderResults = (version, style, width) => (
-    <div className={styles[`nt__${version}`]} style={{ [style]: `${this.state.x}px` }}>
+  renderResults = (graph, styleName, style) => (
+    <div className={styles[styleName]} style={style}>
```
Comment (0xchimz) [inline src/shared/nt-results/ResultsByLoadContainer.react.js]: `key`

**Review by 0xchimz — CHANGES_REQUESTED** [review]

File: src/shared/nt-results/ResultsByLoadContainer.react.js [hunk 4]
```diff
@@ -140,7 +136,8 @@ class ResultsByLoadContainer extends React.Component {
       key='divider'
       className={styles.nt__divider}
       style={{ transform: `translateX(${this.state.x}px)` }}
-      ref={c => { this.divider = c }}
+      onClick={() => { console.log('click') }}
```
Comment (0xchimz) [inline src/shared/nt-results/ResultsByLoadContainer.react.js]: remove `console.log`

**Review by 0xchimz — CHANGES_REQUESTED** [review]

**Review by 0xchimz — APPROVED** [review]

**Pull request merged**